src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx

```diff
@@ -68,14 +68,6 @@ describe('components/PDOKAutoSuggest', () => {
   })
 
   describe('municipality', () => {
-    it('should call fetch without municipality by default', async () => {
-      await renderAndSearch()
-      expect(fetch).toHaveBeenCalledWith(
-        expect.not.stringContaining(municipalityQs),
-        expect.objectContaining({ method: 'GET' })
-      )
-    })
-
```

Comment:
Is this assumption no longer valid?

Reply:
No, the default municipality is always Amsterdam